docker-compose.yaml

```diff
@@ -22,16 +22,28 @@ services:
             COUCHDB_USER: 'couch'
             COUCHDB_PASSWORD: '123'
 
+    prometheus:
+        restart: always
+        image: 'prom/prometheus:v1.7.1'
+        ports:
+            - '9090:9090'
+        volumes:
+            - '/opt/prometheus/:/prometheus'
+            - './etc/prometheus/prometheus-docker-compose.yml:/etc/prometheus/prometheus.yml'
+            - './etc/prometheus/faucet.rules:/etc/prometheus/faucet.rules'
+        links:
+            - faucet
+
     grafana:
         restart: always
-        image: 'grafana/grafana:3.1.1'
+        image: 'grafana/grafana:4.4.1'
         ports:
             - '3000:3000'
         volumes:
             - '/opt/grafana:/var/lib/grafana'
         links:
             - db
```

Comment:
I propose we rename the "db" container to "influxdb" now that we have multiple databases in play.